application/src/main/resources/thingsboard.yml

```diff
@@ -639,6 +639,36 @@ redis:
   db: "${REDIS_DB:0}"
   # db password
   password: "${REDIS_PASSWORD:}"
+  # ssl config
+  ssl:
+    # Enable/disable secure connection
+    enabled: "${REDIS_SSL_ENABLED:false}"
```

Comment:
In other properties below, used TB_REDIS....
Plz use TB_REDIS_SSL_ENABLED (for each yml file)